src/Components/Common/Buttons/ChartSelectionButton/ChartSelectionButton.tsx

```diff
@@ -36,7 +38,7 @@ export const ChartSelectionButton: FC<PropsType> = ({
     // @ts-ignore
     <CustomTag {...props}>
       <img src={src} alt={text} />
-      <p className={classNameText}>{text}</p>
+      {!isInfoPage && <p className={classNameText}>{text}</p>}
```

Comment:
I think it is not a good name for the prop, the "button" component should not know about your pages at all
It makes sense to rename it to smth like "showLabel, showText, showName" or "hideName", so the prop should be "text" specific in your case

Reply:
Done!